packages/debugger-extension/src/index.ts

```diff
@@ -213,7 +217,7 @@ const notebooks: JupyterFrontEndPlugin<IDebugger.IHandler> = {
         }
 
         const { content, sessionContext } = widget;
-        const restarted = await sessionDialogs.restart(sessionContext);
+        const restarted = await sessionDialogs!.restart(sessionContext);
```

Comment:
Looks like we might be able to avoid the `!` by using a different name for the service above?

For example: `const sessionDialogs = sessionContextDialogs ?? new SessionContextDialogs({ translator });`